examples/spawn-and-move/src/actions.cairo

```diff
@@ -129,5 +144,10 @@ mod tests {
         let new_position = get!(world, caller, Position);
         assert(new_position.vec.x == 11, 'position x is wrong');
         assert(new_position.vec.y == 10, 'position y is wrong');
+
+        actions_system.delete_move();
+        // check world state
+        let moves = get!(world, caller, Moves);
+        assert(moves.remaining == 0, 'moves is wrong');
```

Comment:
This test is incorrectly passing as remaining is already zero at this point. 

But for some reason the `delete!` macro seems to be failing (maybe that was ur earlier hunch!). I'll file an issue

Reply:
Oh. Gonna fix it! Thank you so much